index.html

```diff
@@ -2800,7 +2800,7 @@ <h3>Concatenation of strings</h3>
 <aside class="example" title="Example of bidi generation guidance">
 	<p>Here is an example of a Note that you can include in your specification when recommending that implementations generate displayable [=natural language=] values:</p>
 	<div class="note" role="note" id="bidi-gen-note-example">
-	   <p class="example_note">If values generated for <em><code>_field_name_</code></em> contain or might contain [= bidirectional text =], [=producers=] should ensure that [=consumers=] of the string will display it correctly without having to process or alter the string. This can be done using markup or via the insertion of Unicode bidirectional controls. In addition, supplying the correct base direction as metadata allows the [=consumer=] to both isolate the string and set its direction, avoiding [=spillover=] effects.  For more information see: <a href="https://www.w3.org/International/questions/qa-bidi-unicode-controls"><cite>How to use Unicode controls for bidi text</cite></a></p>
+	   <p class="example_note">When generating a value for <em><code>_field_name_</code></em>, [=producers=] need to ensure that any [=bidirectional text=] inside the string will display correctly. When the value is assembled from multiple strings, individual substrings might need to be bidi isolated in order to prevent [=spillover=] effects. This can be done using markup (where available) or via the insertion of Unicode bidirectional controls. In addition, supplying the correct base direction as metadata allows the [=consumer=] to both isolate the string and set its direction, avoiding [=spillover=] effects between the full string and the context in which it is eventually displayed.  For more information see: <a href="https://www.w3.org/International/questions/qa-bidi-unicode-controls"><cite>How to use Unicode controls for bidi text</cite></a> and <a href="#inline_changes">here</a>.</p>
```

Comment:
I'd be inclined to emphasize the word 'inside', just to make it clear.  We could also talk about changes of direction, rather than bidi text, but that's just a thought.

Reply:
> When the value is assembled from multiple strings, individual substrings might need to be bidi isolated in order to prevent [=spillover=] effects.

Do you mean here by the producer or by the consumer?  It's not clear to me.

Think a bit further, i wonder whether introducing something like the producer/consumer bullet points made for SPC would help here?

Reply:
> In addition, supplying the correct base direction as metadata allows the [=consumer=] to both isolate the string and set its direction, avoiding [=spillover=] effects between the full string and the context in which it is eventually displayed.

Here it might help to say something like: In addition, if the producer can provide metadata that indicates the expected base direction for the string as a whole, the consumer will be able to both isolate ....

Reply:
Excellent comments. I think I addressed them in the latest push.